activesupport/lib/active_support/railtie.rb

```diff
@@ -7,6 +7,13 @@ class Railtie < Rails::Railtie # :nodoc:
 
     config.eager_load_namespaces << ActiveSupport
 
+    initializer "active_support.set_authenticated_message_encryption_as_default" do |app|
```

Comment:
Yep, see the other comment about why 😊

Let's just clip "_as_default" from the initializer title while we're here.